locksmith/__tests__/controllers/v2/verifierController.test.ts

```diff
@@ -1,86 +1,122 @@
+import { ethers } from 'ethers'
 import request from 'supertest'
 import { getWalletInput } from '../../test-helpers/utils'
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
 const app = require('../../../src/app')
 
 jest.setTimeout(600000)
-const lockAddress = '0x3F09aD349a693bB62a162ff2ff3e097bD1cE9a8C'
 
-describe('Verifier v2 endpoints for locksmith', () => {
-  it('Add verifier to list and not add duplicates', async () => {
-    expect.assertions(3)
-    const network = 4
+const lockAddress = '0x3F09aD349a693bB62a162ff2ff3e097bD1cE9a8C'
+const managedLock = '0xdCc44A9502239657578cB626C5afe9c2615733c0'
+const network = 4
+jest.mock('@unlock-protocol/unlock-js', () => {
+  return {
+    Web3Service: jest.fn().mockImplementation(() => {
+      return {
+        isLockManager: (lockAddress: string) => lockAddress === managedLock,
+      }
+    }),
+  }
+})
 
-    const {
-      walletAddress: address,
-      message,
-      signedMessage,
-    } = await getWalletInput()
-    const loginResponse = await request(app).post('/v2/auth/login').send({
-      signature: signedMessage,
-      message: message.prepareMessage(),
-    })
-    expect(loginResponse.status).toBe(200)
+jest.mock('../../../src/controllers/v2/verifierController', () => {
+  const mockDispatcher = {
+    list: jest.fn((_req, res) =>
+      res.status(200).send({
+        results: [],
+      })
+    ),
+    addVerifier: jest.fn(() => Promise.resolve()),
+    removeVerifier: jest.fn(() => Promise.resolve()),
+  }
+
+  return jest.fn().mockImplementation(() => {
+    return mockDispatcher
+  })
+})
 
-    const addVerifierResponse = await request(app)
-      .put(`/v2/api/verifier/${network}/${lockAddress}/${address}`)
-      .set('authorization', `Bearer ${loginResponse.body.accessToken}`)
+jest.mock('../../../src/utils/lockManager', () => {
```

Comment:
Same here you don't want to mock the middleware... because you want to make sure it does what you expect it to do.